cmd/ansSendEvent_test.go

```diff
@@ -0,0 +1,127 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/SAP/jenkins-library/pkg/ans"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+const testTimestamp = 1651585103
+
+func TestRunAnsSendEvent(t *testing.T) {
+	t.Parallel()
```

Comment:
is there the need to run in parallel?
not sure, has the code to be thread save?
i'd drop it.

Reply:
done